.github/workflows/node.js.yml

```diff
@@ -15,21 +15,19 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x]
```

Comment:
- I removed 12.x since it has been unsupported for a while
- I tried to add 18.x but restify tests crash immediately, something about a readable stream. We test against an old version of restify so I didn't even try to debug the issue.